settings.php

```diff
@@ -25,23 +25,98 @@
 
 defined('MOODLE_INTERNAL') || die();
 
-if ($hassiteconfig) {
+require_once(__DIR__ . '/lib.php');
 
-    $externalpage = new admin_externalpage('tool_objectfs',
-                                            get_string('object_status:page', 'tool_objectfs'),
-                                            new moodle_url('/admin/tool/objectfs/object_status.php'));
+global $PAGE, $CFG;
 
-    $ADMIN->add('reports', $externalpage);
+if (!$ADMIN->locate('tool_objectfs')) {
 
-    $externalpage = new admin_externalpage('tool_objectfs_settings',
-                                            get_string('pluginname', 'tool_objectfs'),
-                                            new moodle_url('/admin/tool/objectfs/index.php'));
+    $ADMIN->add('tools', new admin_externalpage('tool_objectfs_presignedurl_testing',
+        get_string('presignedurl_testing:page', 'tool_objectfs'),
+        new moodle_url('/admin/tool/objectfs/presignedurl_tests.php')));
 
-    $ADMIN->add('tools', $externalpage);
+    $ADMIN->add('tools', new admin_externalpage('tool_objectfs_object_status',
```

Comment:
Shouldn't these stay / or be moved into the reports admin tab?

Reply:
Yep, didn't catch that, done in latest commit.